### PR TITLE
Sprint16 enable notebook templates

### PR DESCRIPTION
**Pull request by aditee-accenture**

### CODE REVIEW

File: src/notebookTemplates/listNotebookTemplates.tsx [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+
+    app.commands.execute('docmanager:open', {
+      path: filePath
+    });
+  };
+
+  const handleClick = async (template: any) => {
+    const notebookUrl = template.url;
+
+    loggedFetch(notebookUrl, {
```
Comment (Shubha-accenture) [inline src/notebookTemplates/listNotebookTemplates.tsx:86]: Move all the API calls to service files.

File: src/notebookTemplates/listNotebookTemplates.tsx [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+  const tableDataCondition = (cell: ICellProps) => {
+    return (
+      <td {...cell.getCellProps()} className="notebook-template-table-data">
+        {cell.render('Cell')}
+      </td>
+    );
+  };
+
+  const listTemplateAPI = async () => {
+    loggedFetch(NOTEBOOK_TEMPLATES_LIST_URL, {
```
Comment (Shubha-accenture) [inline src/notebookTemplates/listNotebookTemplates.tsx:140]: Move all the API calls to service files.

File: src/notebookTemplates/listNotebookTemplates.tsx [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+    })
+      .then((response: Response) => {
+        if (!response.ok) {
+          throw new Error(
+            `Failed to fetch notebook content. Status: ${response.status}`
+          );
+        }
+        return response.json();
+      })
+      .then((responseData: any) => {
```
Comment (Shubha-accenture) [inline src/notebookTemplates/listNotebookTemplates.tsx:154]: Avoid usage of any. Instead define and interface or type. Look for reference in other API calls.

**Review by Shubha-accenture — CHANGES_REQUESTED** [review]

File: src/notebookTemplates/listNotebookTemplates.tsx (hunk 1 above)
Comment (aditee-accenture) [reply src/notebookTemplates/listNotebookTemplates.tsx]: Moved the calls to service file

**Review by Shubha-accenture — APPROVED** [review]

**Pull request merged**